github-actions/src/main/scala/org/typelevel/sbt/gha/GitHubActionsPlugin.scala

```diff
@@ -120,6 +120,11 @@ object GitHubActionsPlugin extends AutoPlugin {
     }
   }
 
+  def appendtoStepSummary(summaryText: String) =
```

Comment:
Let's make this private for now, we can use it internally in sbt-typelevel, and make it public if there's a usecase / request. This gives us a chance to tweak the API if we want to.
```suggestion
  private[typelevel] def appendtoStepSummary(summaryText: String) =
```

Besides, if someone _really_ wants this its easy to inline in their own plugin.